include/effects/strip/paletteeffect.h

```diff
@@ -129,18 +130,18 @@ class PaletteEffect : public EffectWithId<idStripPalette>
 
         if (_gapSize == 0)
         {
-          for (int i = 0; i < _cLEDs; i+=_lightSize)
+          for (int i = 0; i < LEDStripEffect::_cLEDs; i+=_lightSize)
           {
             iColor = fmodf(iColor + _density, 256);
-            setPixelsOnAllChannels(i, _lightSize, ColorFromPalette(_palette, iColor, 255 * _brightness, _blend), false);
+            LEDStripEffect::setPixelsOnAllChannels(i, _lightSize, ColorFromPalette(_palette, iColor, 255 * _brightness, _blend), false);
```

Comment:
Why is the class name required here?

Reply:
Short answer: because it won't compile without it. :) 

More meaningful version: because it's a _class template_ inheriting from another class template, instead of an actual _class_ doing the same, the compiler requires that any references to functions of superclasses are made explicit. 

There are other ways to do it (`this->` instead of the class name qualifier, or pulling it in with `using`), but I figured it didn't hurt to be specific here.